packages/wallets/src/components/WalletCurrencyCard/WalletCurrencyCard.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { WalletCardIcon } from '../WalletCardIcon';
+import { WalletGradientBackground } from '../WalletGradientBackground';
+import './WalletCurrencyCard.scss';
+
+type TProps = {
+    currency: string;
+    isDemo?: boolean;
+    size?: 'lg' | 'md' | 'sm' | 'xs';
```

Comment:
same here
Right

Reply:
I will remove `lg` type from this component, since i dont think we have currency card larger then medium size